### DISCUSSION TITLE
[@xstate/inspect]: Allow it to work in WebWorker/contexts withouut access to `window`

### DISCUSSION THREAD

**Discussion by kije** [statelyai/xstate#2818] [Ideas]
I recently used xstate inside a WebWorker, where I performed some computation-intensive tasks and used xstate to represent the states of these computations.

However, as I discovered, there is currently no way to use `@xstate/inspect` to inspect xstate-machines running in a WebWorker or any other context where there is no `window` object available.

I think it would be handy if one could inspect the state machines in the dev-tools, even when they are running in a WebWorker.

**Comment by davidkpiano** [comment]
So would you want to use e.g., websockets instead?
